### ISSUE TITLE
FedEx is a legitimate business

### ISSUE THREAD

**Issue by GBT7** [elliotwutingfeng/GlobalAntiScamOrg-blocklist#212]
FedEx.com should be removed from the list, it's not a scam site

**Comment by elliotwutingfeng** [comment]
Yes it is. However, I do not pre-emptively filter out websites (even obviously legitimate ones) from this blocklist in order to keep it consistent with upstream.

Send all false positive reports to <https://globalantiscam.org>
